searchTree/main.c

```diff
@@ -0,0 +1,28 @@
+#include "queryProcessing.h"
+#include "errorCode.h"
+#include "tree.h"
+#include "tests.h"
+#include <stdio.h>
+
+int main(void) {
+    int errorCode = NO_ERRORS;
+    treeTests(&errorCode);
+    Node *root = NULL;
+    int userQuery = -1;
+    while (userQuery) {
```

Comment:
```suggestion
    while (userQuery != 0) {
```
Избегайте неявных кастов